src/Snppts/Snppts.csproj

```diff
@@ -9,11 +9,11 @@
     <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.2.0" PrivateAssets="All" />
     <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="4.4.0" />
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.2.4" />
     <PackageReference Include="X.PagedList.Mvc.Core" Version="7.6.0" />
   </ItemGroup>
 
   <ItemGroup>
-    <Folder Include="Views\Snippets\" />

```

Comment:
Still a line is being removed. And I see you added a new feature now as well? :)

Reply:
I dont know why the visual studio remove this line but i reverted again this change.
Yea man! I add author page, i have change the description to info this change.
I should create a new branch? probably yes, in next time i will do this. ;)

tks!

Reply:
Awesome! Yeah! Love new features, but indeed next time maybe a separate PR for that. But sill much appreciated!